luigi/rpc.py

```diff
@@ -144,35 +146,36 @@ def __init__(self, url='http://localhost:8082/', connect_timeout=None):
         else:
             self._fetcher = URLLibFetcher()
 
-    def _wait(self):
+    def _get_retry_decorator(self):
```

Comment:
Could this be simplified as:
```
@retry(wait=wait_fixed(self._rpc_retry_wait),
             stop=stop_after_attempt(self._rpc_retry_attempts),
             reraise=True,
             after=_retry_logging))
```
and we check the flag in `_retry_logging`?

Reply:
I think this is good suggestion! fixed in a4d46a0